common/xbps-src/shutils/common.sh

```diff
@@ -386,7 +386,7 @@ setup_pkg() {
     esac
 
     # Check if base-chroot is already installed.
-    if [ -z "$bootstrap" -a "z$show_problems" != "zignore-problems" ]; then
+    if [ -z "$bootstrap" -a "z$show_problems" != "zignore-problems" -a "$XBPS_CHROOT_CMD" != "ethereal" ]; then
```

Comment:
@the-maldridge I have removed this check, the side-effect is that we now require all images to be docker containers to have base-chroot installed.